tests/RobotFramework/tests/cumulocity/firmware/firmware_operation_workflow.robot

```diff
@@ -0,0 +1,34 @@
+*** Settings ***
+Resource            ../../../resources/common.resource
+Library             ThinEdgeIO
+Library             Cumulocity
+
+Suite Setup         Custom Setup
+Test Setup          Custom Test Setup
+Test Teardown       Get Logs
+
+Test Tags           theme:tedge_agent
+
+
+*** Test Cases ***
+Send firmware update operation from Cumulocity IoT
+    Cumulocity.Should Contain Supported Operations    c8y_Firmware
```

Comment:
It might also be worthwhile to add a check for the local capability message as well, e.g. `te/device/main///cmd/firmware_update`, to ensure that the functionality on the main device is discoverable to other clients listening to the same MQTT broker.

Reply:
Resolved by bdaabc5